modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js

```diff
@@ -64,7 +64,7 @@ var log = new Log();
         if (log.isDebugEnabled()) {
             log.debug("Request sent to IdP");
         }
-        print("<div><p>You are now being redirected to SSO Provider. If the redirection fails, please click on the "+
+        print("<div><p>You are now being redirected to SSO Provider. If the redirection fails, click on the "+
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158